instaloader/instaloader.py

```diff
@@ -643,11 +643,16 @@ def test_login(self) -> Optional[str]:
     def login(self, user: str, passwd: str) -> None:
         """Log in to instagram with given username and password and internally store session object.
 
-        :raises InvalidArgumentException: If the provided username does not exist.
         :raises BadCredentialsException: If the provided password is wrong.
-        :raises ConnectionException: If connection to Instagram failed.
         :raises TwoFactorAuthRequiredException: First step of 2FA login done, now call
-           :meth:`Instaloader.two_factor_login`."""
+           :meth:`Instaloader.two_factor_login`.
+        :raises LoginException: An error happened during login (for example, and invalid response).
```

Comment:
I do not understand the phrase `for example, and invalid response`. Probably a typo?

Reply:
It was a typo, it should have been 'an invalid responsed'. I've fixed the typo and improved a bit the wording.